src/index.html

```diff
@@ -5,8 +5,339 @@
+    <link 
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="home-link">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo-img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--is-active is-active" href="#apple">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+              href="#laptops&computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <section class="card" data-qa="card">
+        <img
+          class="card__photo"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac (MNED2UA/A)"
+        >
+        <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+      
+        <div class="card__rate">
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+      
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+      
+        <a
+          href="#"
+          class="card__link"
+          data-qa="card-hover"
+        >
+         Buy
+        </a>
+      </section>
+      <section class="card">
```

Comment:
Let's add an empty line between the sibling elements to increase code readability